requirements.txt

```diff
@@ -188,7 +188,7 @@ httpx==0.23.1
     #   shared
 identify==2.5.30
     # via pre-commit
-idna==2.10
+idna==3.7
```

Comment:
Do you know if the next time someone runs pip-compile, if this will regress to 2.10? Or once 3.7, it will persist?

Reply:
Confirmed this wouldn't be the case